modules/op-components.adoc

```diff
@@ -0,0 +1,113 @@
+
+Typically, you configure a complete CI/CD process as a pipeline, while logical parts of this process become tasks.
+
+[id="op-overview-runs_{context}"]
+== Task runs and pipeline runs
+
+`PipelineRun` and `TaskRun` custom resources (CRs) represent individual runs of a pipeline or a task.
+
+{pipelines-shortname} automatically runs a pipeline when you create a `PipelineRun` custom resource referring to an existing pipeline. {pipelines-shortname} automatically runs a task when you create a `TaskRun` custom resource referring to an existing task.
+
+The `PipelineRun` resource sets the variable parameters that the pipeline has. The `TaskRun` resource sets the variable parameters that the task has.
+
+[id="op-overview-triggers_{context}"]
+== Triggers
+
+You can manually create `PipelineRun` and `TaskRun` resources to run pipelines and tasks. Sometimes when certain events, such as a Git repository push happens, you can run a CI/CD pipeline automatically. {pipelines-shortname} provides triggers to support this functionality.
+
+The following components implement a trigger:
+
+* A `TriggerTemplate` CR specifies a “blueprint” for a `PipelineRun` or `TaskRun`, including the parameters that must be set for the pipeline or task.
+* A `TriggerBinding` CR  extracts the fields from an event payload and stores them as parameters that are defined by a `TriggerTemplate` resource.
+* A `Trigger` CR specifies a trigger template and a trigger binding.
+* An `EventListener` Kubernetes object is an addressable sink that runs in the cluster, listens for events as a Webhook on a specified port, and specifies one or more Triggers.
+
+When an `EventListener` object receives an event, {pipelines-shortname} automatically creates a `PipelineRun` or `TaskRun` resource based on the `TriggerTemplate` CR, setting the parameters from the event payload according to the `TriggerBinding` CR specification. The creation of this resource, in turn, causes the pipeline or task to run.
+
+Optionally, you can define an `Interceptor` CR. An interceptor defines the logic that runs every time an event is received through the `EventListener` object, before this event is processed using a trigger. The interceptor can verify, filter, and modify events.
+
+[id="op-overview-catalog_{context}"]
+== Pipelines Catalog, {artifact-hub}, and {tekton-hub}
+
+A _Pipelines catalog_ is a repository of pipelines and tasks. You can download these pipelines and tasks and use them in your processes.
+
+You can use the community-sourced Pipelines catalog indexed by the {artifact-hub} and {tekton-hub} projects.
+
+[id="op-overview-resolvers_{context}"]
+== Resolvers
+
+Use _resolvers_ to retrieve tasks and pipelines from a repository by an identifier. You can reference a task or pipeline using a resolver instead of defining the task or pipeline in your custom resources. When the task or pipeline must be run, {pipelines-shortname} retrieves and runs the task or pipeline.
```

Comment:
```suggestion
With _resolvers_, you can retrieve tasks and pipelines from a repository by using an identifier. You can reference a task or pipeline by using a resolver instead of defining the task or pipeline in your CRs. When the task or pipeline must be run, {pipelines-shortname} retrieves and runs the task or pipeline.
```
Instead of defining the task or pipeline in your CRs, are you referencing a task or a pipeline **that is** using a resolver, or are you referencing a task or a pipeline **by using** a resolver? Do you see how the writing conveys different meanings and is resulting in ambiguity and confusion for the reader?? 

Read both sentences carefully, understand what we need to exactly convey to the reader, and accordingly, rewrite for clarity.